src/Raven.Server/Documents/RetireAttachmentsSender.cs

```diff
@@ -167,7 +193,8 @@ internal async Task<int> RetireAttachments(int batchSize, long maxItemsToProcess
                                 }
                                 else
                                 {
-                                    LogTimeoutIfNeeded(nameof(AttachmentRetireType.PutRetire), key, duration);
+                                    if (Logger.IsInfoEnabled)
```

Comment:
are we still using dedicated threads (with the costly setup) for each attachment?

Reply:
you sure this is an Info message? This is turned on by default.

```
Log levels ordered by severity:
- Trace (Ordinal = 0) : Most verbose level. Used for development and seldom enabled in production.
- Debug (Ordinal = 1) : Debugging the application behavior from internal events of interest.
- Info (Ordinal = 2) : Information that highlights progress or application lifetime events.
- Warn (Ordinal = 3) : Warnings about validation issues or temporary failures that can be recovered.
- Error (Ordinal = 4) : Errors where functionality has failed or Exception have been caught.
- Fatal (Ordinal = 5) : Most critical level. Application is about to abort.
```

Reply:
@ayende yes, but Egor is making changes gradually - its on a list

Reply:
> are we still using dedicated threads (with the costly setup) for each attachment?

yes will be done here:
https://issues.hibernatingrhinos.com/issue/RavenDB-24606/Retire-attachments-Server-Side-Use-Task-instead-of-Thread-in-DirectFileUploader.cs-Phase-1

for now I tried to stick with minimal changes to finish the actual task in RavenDB-24678